xtask/src/run_wasm.rs

```diff
@@ -9,25 +9,19 @@ pub(crate) fn run_wasm(shell: Shell, mut args: Arguments) -> anyhow::Result<()>
     let no_serve = args.contains("--no-serve");
     let release = args.contains("--release");
 
-    let programs_needed: &[_] = if no_serve {
-        &[Program {
-            crate_name: "wasm-bindgen-cli",
-            binary_name: "wasm-bindgen",
-        }]
-    } else {
-        &[
-            Program {
-                crate_name: "wasm-bindgen-cli",
-                binary_name: "wasm-bindgen",
-            },
-            Program {
-                crate_name: "simple-http-server",
-                binary_name: "simple-http-server",
-            },
-        ]
-    };
-
-    check_all_programs(programs_needed)?;
+    let mut programs_needed = vec![Program {
+        crate_name: "wasm-bindgen-cli",
+        binary_name: "wasm-bindgen",
+    }];
+
+    if no_serve {
```

Comment:
This seems wrong. Shouldn't this be inverted (i.e., `!no_serve` should pull in `simple-http-server`, not the inverse)?

Reply:
Oops!